SpectralLineRequest.cc

```diff
@@ -0,0 +1,163 @@
+#include "SpectralLineRequest.h"
+
+#include <curl/curl.h>
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "Table/Columns.h"
+
+using namespace carta;
+
+#define REST_FREQUENCY_COLUMN_INDEX 2
+
+const std::string SpectralLineRequest::SplatalogueURL =
+    "https://www.cv.nrao.edu/php/splat/"
```

Comment:
I wonder if we should build the query string from a base URL and a number of required columns. Might be much easier to understand what's going on than a long URL. In future we may want to be able to customise the base URL for the server build, if it is hosted locally